compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -282,6 +288,38 @@ private void visitBuiltinFunctions(BLangPackage pkgNode, BLangFunction function)
         }
     }
 
+    private void visitMockFunctions(BIRPackage birPkg, Map<String, String> mockFunctionMap) {
```

Comment:
I have used this name since there was another function called `visitBuiltInFunction()` in the same class. Im currently not sure what to rename this function. Any suggestions?

Reply:
Heres a tip for function naming/designing :) 

Try to explain the logic in the code using a short single sentence. e.g: "This method **adds wheels to a car.**"
Then the method signature can be something like `addWheels(Car c)`.
Now, anyone who looks at the method-signature can get a high level idea of what it does, without looking at the code.

Reply:
Since we are replacing the call.Terminator's name and calleepkg details i decided on 'replaceMockFunctions()' as a name. Is this suitable?

Reply:
+1
should it be `replaceMockedFunctions()`?

Reply:
yes. Since we are replacing the functions to be mocked... 

Changed it to `replaceMockedFunctions()`